README.md

```diff
@@ -41,6 +41,14 @@ lmc_cqrs:
         solr: false         # Whether should solr extension be active (requires a lmc/cqrs-solr dependency) [default false]
 ```
 
+### Profiler extended configuration
+```yaml
+lmc_cqrs:
+    profiler:
+        enabled: false      # Whether to enable profiler and allow profiling queries and commands [default false]
```

Comment:
Should it be configured via symfony dev mode? E.g. if symfony aplication is in dev mode, profiler should be always on.

Reply:
I mention this as a tip in the README
```
**TIPs**:
- it is advised to set `profiler: '%kernel.debug%'` ...
```

Yet you need to do more than just enable it (you have to manually register routes for CQRS bundle, so it could crush without you knowing it is enabled.